app/jobs/update_sample_metadata_job.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateSampleMetadataJob < ApplicationJob
+  queue_with_priority 1
+  queue_as QueueNames::SAMPLES
+  def perform(sample_type)
+    Seek::Samples::MetadataUpdater.new(sample_type).update_metadata
```

Comment:
this is going to need some profiling and scalability testing with a sample type linked to large set of samples. I'll get some metrics from fairdomhub about what is a our current maximum